modules/apps/frontend-theme/kyoung-liferay-theme/src/js/canvas.js

```diff
@@ -0,0 +1,89 @@
+var space;
+
+function floatySpace() {
+    var colors = [
+        '#FF3F8E', '#04C2C9', '#2E55C1'
+    ];
+
+    space = new CanvasSpace('canvas', '#252934').display();
+    var form = new Form(space);
+
+	// Elements
+
+    var pts = [];
+    var center = space.size.$divide(1.8);
+    var angle = -(window.innerWidth * 0.5);
+	var count = window.innerWidth * 0.05;
+
+    if (count > 150) count = 150;

```

Comment:
JS `if/else` statements should be formatted with curly braces and spaced out so it can be easily read.
Ex.
```
if (x > z) {
    ...
}
```